README.md

```diff
@@ -1,20 +1,28 @@
 # JQuery/Ajax Quiz
 
-Write your answer below each question. 
+Write your answer below each question.
 
 ### Question 1
-(a) What is a click handler? 
+(a) What is a click handler?
 
-(b) Where in your JS file should you put it, and WHY? 
+A click handler is an event handler that fires when the listener hear a 'click'.
```

Comment:
More accurate to say a click handler is an EVENT that fires when the listener hears a 'click.'